aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -195,9 +303,13 @@ Resources:
       AutoScalingGroupName: {Ref: WebServer}
       MinSize: 2
       Recurrence: "0 3 * * TUE-SAT"
+<%   end -%>
```

Comment:
This property of CloudFormation allows us be confident that pushing a single change within the stack template won't inadvertently affect other unrelated running resources. (And we can confirm what would be changed by updating to the current template in advance, running `RAILS_ENV=[environment] rake stack:validate`.)